File: basic_math/primenumber/primenumber.py
n=int(input("Enter:"))
isprime=True
if(n<=1):
   isprime=False
else:
   for i in range(2,int(n+1/2)):
      if(n%i==0):
         isprime=False
         break
if isprime:
   print("prime")
else:
   print("not prime")

File: basic_math/primenumber/primenumber.js
let n=5;
let isprime=true;
while(n<=1){
    isprime=false;
}
for(let i=2;i<=n/2;i++){
    if(n%i==0){
        isprime=false;
        break;
    }

}
if(isprime){
    console.log("prime");

}
else{
    console.log("not prime");
}